db/migrations/0018_add_lti_usage.py

```diff
@@ -0,0 +1,43 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement_types (
+            id INTEGER NOT NULL AUTO_INCREMENT,
+            description VARCHAR(100) NOT NULL,
+            table_name VARCHAR(64) NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        INSERT IGNORE INTO lti_placement_types
```

Comment:
Is "ignore" really needed here?

Reply:
I only left it in there because we have a pattern of `CREATE TABLE IF NOT EXISTS`. We wouldn't want to insert twice.

The migrations would prevent this from running so we *really* don't need the INSERT IGNORE or IF NOT EXISTS.

Reply:
I'm just deleting this table and doing it in the code